src/Entities/Project.php

```diff
@@ -37,6 +37,11 @@ public function getIsOverdue(): ?bool
         return $this->isOverdue;
     }
 
+    public function getDeadline(): ?DateTime
```

Comment:
need a comment to say this is only being used for unit testing

Reply:
Done